### ISSUE TITLE
Link broken

### ISSUE THREAD

**Issue by didrikmunther** [datasektionen/bawang-content#139]
The link "Uppflyttningskrav" is broken in the sidebar file:

https://github.com/datasektionen/bawang-content/blob/master/studier/sidebar.md

**Comment by triggerdude33** [comment]
Yes, it seems that "uppflyttningskrav" is no longer a thing as of 2018. From what I've gathered, "uppflyttningskrav" was basically that you had to read x amount of HP in order to advance to the next year's courses.
https://www.kth.se/student/kurser/program/CDATE/20182/genomforande
It's been replaced by "särskild behörighet" for taking courses. In other words, that you for example have to have passed envarre in order to take the ADK course. And these new regulations are determined on a course-by-course basis. So I'm not sure we can find a good replacement for this link. Maybe if we can find an overview of how all the courses in the programme depend on each other.
